config_script/process_config.py

```diff
@@ -177,7 +182,21 @@ def add_firebase_config(self, config, firebase_info_plist_path):
             self.plist_manager.write_to_plist_file(plist, self.plist_manager.get_firebase_info_plist_path())
         else:
             print("Firebase config is empty. Skipping")
-
+   
+    def add_branch_config(self, config, plist):
+        branch = config.get('BRANCH', {})
+        enabled = branch.get('ENABLED')
+        uriScheme = branch.get('URI_SCHEME')
```

Comment:
I don't see `URI_SCHEME` inside `BRANCH` in config - is it expectable and will be added later?

Reply:
Ah ok - found this in PR Description
Resolved